service/collector/elb.go

```diff
@@ -139,31 +143,75 @@ func (e *ELB) collectForAccount(ch chan<- prometheus.Metric, awsClients clientaw
 
 	var lbs []loadBalancer
 	{
-		i := &elb.DescribeTagsInput{}
-		for _, l := range loadBalancers {
-			i.LoadBalancerNames = append(i.LoadBalancerNames, l.LoadBalancerName)
+		// AWS API has a limit for maximum number of LoadBalancerNames in
+		// single Describe request so it must be done in batches of
+		// maxELBsInOneBatch. In order to not spend so much time on this,
+		// perform requests concurrently and synchronize them with errgroup.
+		errGroup, _ := errgroup.WithContext(context.Background())
+		tagOutputs := make([]*elb.DescribeTagsOutput, len(loadBalancers)/maxELBsInOneDescribeTagsBatch+1)
```

Comment:
A comment for the math here would be helpful, I don't follow the why

Reply:
Comment could be helpful but it's just calculating the number of output badges we expect.

Reply:
Example: when you have 42 load balancers, and you process them by batches of 20. You'll have 3 batches.

Hence:
```go
a := len(loadBalancers)            // 42
b := maxELBsInOneDescribeTagsBatch // 20
c := a/b+1                         // 3
```

Reply:
Exactly. Added also comment telling the same. :slightly_smiling_face:

Reply:
That's the best microoptimization I've seen in a while :D

Reply:
Ack. It's no longer necessary when using `append`.